_includes/manuals/1.7/5.2.9_cr_vmware.md

```diff
@@ -68,7 +68,19 @@ cp /vmfs/volumes/datastore1/vnc.xml /etc/vmware/firewall/
 esxcli network firewall refresh
 {% endhighlight %}
 
-
+#### Required Permissions
+The minimum permissions to properly provision new virtual machines is:
```

Comment:
I'm not a native English speaker, but should it be "The minimum permissions ... are:"?